libflux/src/flux/semantic/builtins.rs

```diff
@@ -124,7 +124,14 @@ pub fn builtins() -> Builtins<'static> {
             "influxdata/influxdb" => maplit::hashmap! {
                 // This is a one-or-the-other parameters function
                 // https://github.com/influxdata/flux/issues/1659
-                "from" => "forall [t0, t1] (?bucket: string, ?bucketID: string) -> [{_measurement: string | _field: string | _time: time | _value: t0 | t1}]",
+                "from" => r#"forall [t0, t1] (
+                    ?bucket: string,
+                    ?bucketID: string,
+                    ?org: string,
+                    ?orgID: string,
+                    ?host: string,
+                    ?token: string
```

Comment:
I don't really understand this change. Why are we now able to accept these additional optional parameters? Seems like a big change unless I'm missing something. Are org and ordID mutually exclusive here like they are in `to()`.

Reply:
Yes, both org and orgID are mutually exclusive. The `org` is needed so that we can specify the org in the flux binary to retrieve from because flux isn't org-aware the way influxdb is, but influxdb requires the organization. Inside of influxdb, the `org` will get filled in automatically.

The reason for `orgID` is because the influx cli can accept an orgID instead and so we need to send that instead for the influxdb repl.